ResetPasswordBundle/src/tests/UnitTests/Persistence/PWResetRequestRepositoryTraitTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SymfonyCasts\Bundle\ResetPassword\tests\UnitTests\Persistence;
+
+use SymfonyCasts\Bundle\ResetPassword\Persistence\PasswordResetRequestRepositoryInterface;
+use SymfonyCasts\Bundle\ResetPassword\tests\Fixtures\PWResetRequestRepositoryTraitTestFixture;
+use PHPUnit\Framework\TestCase;
+use SymfonyCasts\Bundle\ResetPassword\Persistence\Repository\PasswordResetRequestRepositoryTrait;
+
+class PWResetRequestRepositoryTraitTest extends TestCase
+{
+    public function methodDataProvider(): \Generator
+    {
+        yield ['persistPasswordResetRequest'];
+        yield ['findPasswordResetRequest'];
+        yield ['getMostRecentNonExpiredRequestDate'];
+        yield ['removeResetRequest'];
+    }
+
+    /**
+     * @test
+     * @dataProvider methodDataProvider
+     */
+    public function hasMethod(string $method): void
+    {
+        self::assertTrue(method_exists(PasswordResetRequestRepositoryTrait::class, $method));
+    }
```

Comment:
Not needed.

Reply:
removed - used early on when defining class structure